Birthday-2024-Project/Scripts/PieceSetup.gd

```diff
@@ -16,11 +16,24 @@ static func JsonParse(jsonData : String) -> PieceSetup:
 	var error = json.parse(jsonData)
 	if(error == OK):
 		var parsedDictionary : Dictionary = json.data 
+		
+		if parsedDictionary.has(ID_KEY) == false:
```

Comment:
This needs some work. I've removed bold piece name from JSON (["{\"PIECE_SETUP_GRID_POSITION_X_KEY\":0,\"PIECE_SETUP_GRID_POSITION_Y_KEY\":-1,\"PIECE_SETUP_ID_KEY\":\"**casual_fauna_wave**\",\"PIECE_SETUP_ROTATION_KEY\":0}"]) and it clicking import doesn't return error but goes straight to code and crashes the game

Reply:
Now checks the resulting piece ID to make sure it exists, in case it successfully parsed a bad ID